draft-ietf-quic-transport.md

```diff
@@ -2483,8 +2489,8 @@ after a frame bearing the FIN flag is sent.
 
 A stream is "half-closed (remote)" when the stream is no longer being used by
 the peer to send any data.  An endpoint will have either received all data that
-a peer has sent or will have received a RST_STREAM frame and discarded any
-received data.
+a peer has sent, will have sent an UNI flag on the stream, or will have
```

Comment:
I guess this is more of a meta question, at a high level do we need to negotiate unidirectionality on a stream level or would it be sufficient to apply it at a connection level.